# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/Makefile.win32
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/Makefile.win32
--- a/gfx/cairo/cairo/src/Makefile.win32	Mon May 06 19:22:35 2024 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,28 +0,0 @@
-top_srcdir = ..
-include $(top_srcdir)/build/Makefile.win32.common
-include Makefile.win32.features
-
-SOURCES = $(enabled_cairo_sources)
-
-STATIC_SOURCES = cairo-system.c
-
-OBJECTS = $(patsubst %.c, $(CFG)/%.obj, $(SOURCES))
-OBJECTS_STATIC = $(patsubst %cairo-system.obj, %cairo-system-static.obj, $(OBJECTS))
-
-static: inform $(CFG)/cairo-static.lib
-dynamic: inform $(CFG)/cairo.dll
-
-$(CFG)/cairo.dll: $(OBJECTS)
-	@$(LD) $(CAIRO_LDFLAGS) -DLL -OUT:$@ $(CAIRO_LIBS) $(PIXMAN_LIBS) $(OBJECTS)
-
-$(CFG)/cairo-static.lib: $(OBJECTS_STATIC)
-	@$(AR) $(CAIRO_ARFLAGS) -OUT:$@ $(PIXMAN_LIBS) $(OBJECTS_STATIC)
-
-all: inform $(CFG)/cairo.dll $(CFG)/cairo-static.lib
-	@echo "Built successfully!"
-	@echo "You should copy the following files to a proper place now:"
-	@echo ""
-	@echo "	src/cairo-features.h"
-	@for x in $(enabled_cairo_headers); do echo "	src/$$x"; done
-	@echo "	src/$(CFG)/cairo.dll"
-	@echo "	src/$(CFG)/cairo-static.lib"